src/data/countable/defs.lean

```diff
@@ -76,8 +76,7 @@ let ⟨n, ⟨e⟩⟩ := finite.exists_equiv_fin α in countable.of_equiv _ e.sym
 
 instance : countable punit.{u} := subsingleton.to_countable
 
-@[nolint instance_priority]
-instance Prop.countable (p : Prop) : countable p := subsingleton.to_countable
+@[priority 100] instance Prop.countable (p : Prop) : countable p := subsingleton.to_countable
```

Comment:
I used the default priority because this instance always succeeds (assuming `p : Prop`), so this looks like a false positive of the linter to me.